CHALLENGE.md

```diff
@@ -94,9 +95,35 @@ You are given a prepared SPA project that was bootstrapped with Create React App
 ![create-tournament-prompt](.github/create-tournament-prompt.png)
 
 - Clicking on the `CREATE TOURNAMENT` button will open a browser prompt with a message `Tournament Name:`, an input field to enter the tournament name and the buttons `Cancel` to cancel and `OK` to confirm.
-- When confirming, the tournament will be created on the fake REST API and added on the UI without any loading indicators.
+- The tournament name must contain only Latin letters, numbers, and spaces, not an empty string or only spaces.
+- When confirming, the tournament will be created immediately using an optimistic update in the UI and a fetch call on the fake REST API.
```

Comment:
if I'm not mistaken we don't expect the candidates to implement optimistic update, when new tournament is created, only when edited or deleted.

Reply:
then it's not consistent
also, it's the same logic, I don't believe that will take more than 5 minutes to implement if you already have optimistic updates for edit/delete in place.

Reply:
Here are more details on that https://faceit.slack.com/archives/GMSMHQAQJ/p1657551725576399 with the point from Michele, why we don't and should not ask about it

Reply:
I understand could be not consistence , but could lead other problems, such the id not exist (then using different key when using map), or add a template data. Will keep simple only Edit and Delete, and continue how it was.